cli/flox-rust-sdk/src/models/search.rs

```diff
@@ -290,42 +299,123 @@ impl TryFrom<&[u8]> for SearchResults {
 }
 
 /// Calls `pkgdb` and reads a stream of search records.
+#[allow(clippy::type_complexity)]
```

Comment:
commentary (NB):`std::thread::Result<Result<(Vec<SearchResult>, Option<u64>), SearchError>>` is already too complex for clippy?

Reply:
Yep :)